modules/apps/redirect/redirect-service/src/main/java/com/liferay/redirect/internal/provider/RedirectProviderImpl.java

```diff
@@ -156,6 +160,52 @@ protected void setRedirectPatternEntries(
 		_redirectPatternEntries = redirectPatternEntries;
 	}
 
+	private String[] _getCrawlerUserAgents() {
+		return new String[0];
+	}
+
+	private boolean _isCrawlerUserAgent(String userAgent) {
+		for (String crawlerUserAgent : _getCrawlerUserAgents()) {
+			if (userAgent.contains(StringUtil.toLowerCase(crawlerUserAgent))) {
+				return true;
+			}
+		}
+
+		return false;
+	}
```

Comment:
about the name, the same as the previos message. the actual configuration is `crawlerUserAgents` so I think that the name should be this.

On the other case, each `crawlerUserAgent` comes from the configuration and we can not know if is lower, upper o camel, so it has to be done here.

Reply:
On another hand, if we left the _getCrawlerUserAgents method, we can do the lower case there, but I prefer to left this till that moment because I do not know right now if _getCrawlerUserAgents will be erased and we only will call the configuration or is better to modify the values of the configuration.
🤷🏻‍♀️

Reply:
I think it's to over complicate the solution (managed services + new configuration vs only new configuration) only to do not make the lowercase

My final idea is to make this call:
`_isCrawlerUserAgent(_newConfiguration.crawlerUserAgents(), userAgent)`

as in : 
https://github.com/liferay/liferay-portal/blob/499cd9a8c9c3528fecb92f536c985c3df4bf8da6/modules/apps/layout/layout-seo-web/src/main/java/com/liferay/layout/seo/web/internal/servlet/filter/LayoutSEODynamicRenderingFilter.java#L69

That is the configuration we will change in the future.

That maybe is worth but.... 

I prefer to talk about this an the other suggestion on a meeting because I have my doubts 

Thanks

Reply:
As discussed in Slack, we'll keep things as they are for the moment. In future PRs will revisit the crawler configuration to make it more efficient.